src/benchmark/benchmark.ts

```diff
@@ -72,7 +73,15 @@ class Benchmark {
         Benchmark.executeJS(func[0], args, expectedResult, warmupRounds);
         const jsTimes = Benchmark.executeJS(func[0], args, expectedResult, measureRounds);
 
-        const callWrapper = transpiler.transpile(func.join(''), signatures);
+        for (const name of signatures.keys()) {
```

Comment:
You could use `Map.entires()` here, this would save the `Map.get()` call and the undefined check as well.